snippets/chunk-array.md

```diff
@@ -0,0 +1,14 @@
+### Chunk Array
+
+Creates an array of elements split into groups the length of size. 
+If array can't be split evenly, the final chunk will be the remaining elements.
+
+```js
+const chunk = (arr, size) => 
+  Array
+    .apply(null, {length: Math.ceil(arr.length/size) })
+    .map((value, index) => arr.slice(index*size, index*size+size) )
+
+// const myArray = [2, 2, 2, 2, 2, 2, 3, 2, 3, 2, 3, 2, 2];
+// chunk(myArray, 3) -> [ [ 2, 2, 2 ], [ 2, 2, 2 ], [ 3, 2, 3 ], [ 2, 3, 2 ], [ 2 ] ]
```

Comment:
The example can be reduced to one line and shortened (fewer elements, chunks of `size = 2` should be fine).

Reply:
Thanks 😄! Sorry for the late reply, went out with the family for the holidays. I will be sure to follow the recommended format with my next pull request 👍